### DISCUSSION TITLE
Add support for Heroicons v2.0.0 or other icons via SVG

### DISCUSSION THREAD

**Discussion by wamesro** [laravel/nova-issues#4745] [Ideas & Feature Requests] [answered]
A new version of heroicons 2.0.0 has been released https://heroicons.com/
are you planning to add this package support for item menu?

or it would be better to add your own SVG that could be copied either from Heroicons or e.g. https://icons.getbootstrap.com/

**Comment by crynobone** [comment, marked as answer]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
